src/components/CommandBar/CommandBarHeader.tsx

```diff
@@ -102,19 +105,22 @@ function CommandBarHeader({ children }: React.PropsWithChildren<object>) {
                 <span className="pr-2" />
               )}
             </p>
-            {Object.entries(nonHiddenArgs || {})
-              .filter(
-                ([_, argConfig]) =>
-                  argConfig.skip === false ||
-                  (typeof argConfig.required === 'function'
-                    ? argConfig.required(commandBarState.context)
-                    : argConfig.required)
-              )
-              .map(([argName, arg], i) => {
+            {Object.entries(nonHiddenArgs || {}).flatMap(
```

Comment:
Moved to flatMap so that we wouldn't have to compute `argValue` twice here for the filtering needs. We actually want to show non-hiddent optional args that have a value set already